modules/plottinggl/src/processors/parallelcoordinates/parallelcoordinates.cpp

```diff
@@ -571,7 +575,7 @@ void ParallelCoordinates::drawLines(size2_t size) {
     lineShader_.setUniform("selected", 0);
     lineShader_.setUniform("filtered", 0);
     for (size_t i = 0; i < numLines; i++) {
-        if (brushingAndLinking_.isFiltered(indexCol[i])) {
+        if (brushedID.find(indexCol[i]) != brushedID.end()) {
```

Comment:
How will this affect filtering from other components? Should it not be brushingAndLinking_.isFiltered(indexCol[i]) || brushedID.find(indexCol[i]) != brushedID.end() ?